README.md

```diff
@@ -126,7 +126,8 @@ Checks if the value of a graphite metric has received data recently.
 #### `cloudWatchThreshold`
 Checks whether the value of a CloudWatch metric has crossed a threshold
 
-_Note: this assumes that `AWS_ACCESS_KEY` & `AWS_SECRET_ACCESS_KEY` are implictly available as environment variables on process.env_
+_Note: this assumes that `AWS_ACCESS_KEY` & `AWS_SECRET_ACCESS_KEY` are implicitly main
```

Comment:
question: what does 'main' mean in this context?

Reply:
Another typo, fix thanks